Gruntfile.js

```diff
@@ -102,6 +102,9 @@ module.exports = function(grunt) {
         }, {
           from: '$DIM_FLAVOR',
           to: 'release'
+        }, {
+          from: '$DIM_API_KEY',
+          to: '8f0be500571741738b5d0832246a523c'
```

Comment:
if we want to remove the keys entirely from the app, we could instead just pass the keys in when we/travis runs the build task.